### PR TITLE
corrected number

### PR DESCRIPTION
**Pull request by michcada**
corrected number

### CODE REVIEW

**Comment by mhiliri** [comment]
@michcada please remove the unnecessary files

File: .vs/VSWorkspaceState.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (mhiliri) [inline .vs/VSWorkspaceState.json]: This file is not part of the repo

**Comment by michcada** [comment]
I added them to the gitignore is that way ok?

**Comment by mhiliri** [comment]
> I added them to the gitignore is that way ok?

Well, for this repo we don't need .gitignore and the other files and folders. In this Pull Request the only file should be  index.html
Maybe you were doing some testing and accidentally added those files.
Please remove the unnecessary files and folders and commit your changes

**Review by mhiliri — APPROVED** [review]
Nice, Thanks

**Pull request merged**